examples/create-mcs-db-database/main.tf

```diff
@@ -1,12 +1,32 @@
-resource "mcs_db_instance" "test" {
-  region    = "RegionOne"
-  name      = "test"
-  flavor_id = "25ae869c-be29-4840-8e12-99e046d2dbd4"
+terraform {
+  required_providers {
+    mcs = {
+      source = "MailRuCloudSolutions/mcs"
+      version = "0.2.7"
+    }
+    openstack = {
+      source = "terraform-provider-openstack/openstack"
+    }
+  }
+}
+
+data "openstack_compute_flavor_v2" "db" {
+  name = var.db-instance-flavor
+}
+
+resource "openstack_networking_network_v2" "db" {
+  name           = "db-net"
+  admin_state_up = true
+}
+
+resource "mcs_db_instance" "db-instance" {
```

Comment:
please sort keys to the order of reference

Reply:
Fixed